README.md

```diff
@@ -69,7 +69,6 @@ Do for example:
       echo "WRONG CMSSW VERSION"
       return 1
     fi
-    git clone https://gitlab.cern.ch/kit-cn-cms-public/RecoLikelihoodReconstruction.git RecoLikelihoodReconstruction
```

Comment:
Is this really completely deprecated now?

Reply:
Hello sebwieland,
thank you for your comment.

At the moment this module is not used anymore at all, as the Chi2 reconstruction was moved to the CommonClassifier itself.
If we want to activate an likelihood based reconstruction in the future, we need to re-add this module though, as this is not yet implemented in the CommonClassifier.